src/ScottPlot4/ScottPlot/Drawing/GDI.cs

```diff
@@ -159,7 +159,9 @@ public static System.Drawing.Graphics Graphics(Bitmap bmp, PlotDimensions dims,
         {
             Graphics gfx = Graphics(bmp, lowQuality, dims.ScaleFactor);
 
-            if (clipToDataArea)
+            bool dataIsSmallerThanFigure = dims.DataWidth < dims.Width && dims.DataHeight < dims.Height;
```

Comment:
Should this not be `dims.DataWidth < dims.Width || dims.DataHeight < dims.Height`?

We need to clip if at least one dimension is smaller, not just if both are smaller.

Reply:
Thank you for the careful review @bclehmann! 

I updated the code to more clearly communicate what the intent was (only skip clipping for frameless plots)